### PR TITLE
LPS-145704 + LPS-145752 Langauge Override Portlet fixes

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-145704](https://issues.liferay.com/browse/LPS-145704).
This is an update for [LPS-145752](https://issues.liferay.com/browse/LPS-145752).

/cc @pei-jung @alee8888 @ChrisKian @patriciajeanperez @lr-leo @erickmonteiroo

### CODE REVIEW

**Comment by drewbrokke** [comment]
@pei-jung for the JS in LPS-145752, I was hoping to avoid using AUI at all, but using only a native form submit handler did not properly stop form submission.

File: modules/apps/portal-language/portal-language-override-web/src/main/resources/META-INF/resources/edit_plo_entry.jsp [hunk 1]
```diff
@@ -171,4 +171,50 @@ renderResponse.setTitle(editDisplayContext.getPageTitle());
...
+		var auiForm = getForm();
+
+		if (auiForm) {
+			registerOnSubmit(auiForm);
+		}
+		else {
+			Liferay.once('<portlet:namespace />formReady', () => {
+				registerOnSubmit(getForm());
+			});
+		}
```
Comment (pei-jung) [inline modules/apps/portal-language/portal-language-override-web/src/main/resources/META-INF/resources/edit_plo_entry.jsp:218]: We can avoid registering the onSubmit event manually by utilizing the [onSubmit](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/liferay-frontend.tld#L244-L249) attribute in the `<liferay-frontend:edit-form>` tag.

File: modules/apps/portal-language/portal-language-override-web/src/main/resources/META-INF/resources/edit_plo_entry.jsp [hunk 2]
```diff
@@ -171,4 +171,50 @@ renderResponse.setTitle(editDisplayContext.getPageTitle());
...
+				for (const input of inputs) {
+					if (
+						input.name.startsWith('<portlet:namespace />value') &&
+						!!input.value
+					) {
+						return;
+					}
+				}
+
+				event.halt();
```
Comment (pei-jung) [inline modules/apps/portal-language/portal-language-override-web/src/main/resources/META-INF/resources/edit_plo_entry.jsp:193]: Have you tried calling the `event.preventDefault()` before the form submit handler like [this](https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-instances/portal-instances-web/src/main/resources/META-INF/resources/edit_instance.jsp#L42)?

**Comment by drewbrokke** [comment]
https://github.com/liferay-usm/liferay-portal/pull/418

**Pull request closed without merging**